fero/client.py

```diff
@@ -355,3 +369,173 @@ def end_impersonation(self):
         self._fero_token = self._admin_token
         self._admin_token = None
         self._impersonated = None
+
+
+class UnsafeFeroForScripting(Fero):
+    """SCRIPT USE ONLY: An unsafe client for scripting purposes."""
+
+    def _wait_until_datasource_is_ready(self, ds: DataSource) -> DataSource:
+        backoffs = [1, 1, 1, 10, 10, 10, 10, 10, 10, 10, 10, 10]
```

Comment:
It's not inconceivable we may need a larger back off here.   Maybe something like this? https://pypi.org/project/backoff/